plugins/woocommerce-blocks/tests/e2e/tests/templates/cart-template.block_theme.spec.ts

```diff
@@ -27,11 +27,14 @@ test.describe( 'Test the cart template', () => {
 		editor,
 		page,
 	} ) => {
-		await admin.visitSiteEditor( { path: '/page' } );
+		await admin.visitSiteEditor( { postType: 'page' } );
 		await editor.page
 			.getByRole( 'button', { name: 'Cart', exact: true } )
 			.click();
-		await editor.enterEditMode();
+		await page
+			.frameLocator( 'iframe[name="editor-canvas"]' )
+			.locator( 'body' )
+			.click();
```

Comment:
As above, let's use canvas here as well:

```suggestion
		await editor.canvas.locator( 'body' ).click()
```

Reply:
Updated in e15c594a21ff149034c2e322de62ceb652868782.